src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx

```diff
@@ -113,7 +127,17 @@ class FollowArtistPopoverRow extends React.Component<Props, State> {
 }
 
 export const FollowArtistPopoverRowFragmentContainer = createFragmentContainer(
-  FollowArtistPopoverRow,
+  (props: Props) => {
```

Comment:
This is just the convenience pattern to pass the state object in as a prop (`excludeState`) to avoid nesting the entire `render()`, inside a `<Subscribe....`

Reply:
Oh cool! Year, I tried to read about this whole Provider/Subscribe pattern and got as far as creating State object. But got confused about who and where should subscribe to it and update it.